health/notifications/health_alarm_notify.conf

```diff
@@ -1121,3 +1162,4 @@ role_recipients_rocketchat[sitemgr]="${DEFAULT_RECIPIENT_ROCKETCHAT}"
 
 role_recipients_sms[sitemgr]="${DEFAULT_RECIPIENT_SMS}"
 
+role_recipients_dynatrace[sitemgr]="${DEFAULT_RECIPIENT_DYNATRACE}"
```

Comment:
Hi @illumine ,

I could test your PR now, and appears to be working, for I finish the tests, firstly I will ask you to add a new line in the end of this file, as you can see `github` is reporting an error here.